README.md

```diff
@@ -55,10 +55,30 @@ You likely won't need to run `plasma-harness` in production.
 ### 1. Checkout, install deps, and build:
 
 ```bash
+# install go from (https://golang.org/dl/)
+go env GOPATH # should be ~/go
+mkdir ~/go
+mkdir ~/go/bin
+mkdir ~/go/src
+
+export PATH=$PATH:/Users/kirushanth/go/bin
+
+# install dep
+curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+# install blockchain tooling
+[sudo] npm install -g truffle@5.0.3
+[sudo] npm install -g ganache-cli
+
+# install jq (https://stedolan.github.io/jq/download/)
+brew install jq # Mac / OSX
+sudo apt-get install jq # Linux/Ubuntu
+
 mkdir -p $GOPATH/src/github.com/kyokan
 git clone https://github.com/kyokan/plasma.git
 cd plasma
-make deps
+make deps # installs git-submodules / go dependencies / node dependencies
+make abigen # compiles smart-contracts via truffle
```

Comment:
@mslipper the commands right after complained about lacking contracts, so we need to compile the contracts beforehand

Reply:
Ah cool, many thanks!